Makefile

```diff
@@ -21,7 +21,7 @@ site: deps
 	npm run build
 
 publish: all_dists site
-	ipfs add -q -r releases | tail -n1 >>versions
+	ipfs add -q -r releases | tail -n1
```

Comment:
Actually, I found it useful to have a file which tracks which releases ive built.

wanna do `ipfs add -q -r releases | tee -a versions` ?

Reply:
I agree with @hsanjuan

Reply:
Sure (but I'm going to call it something else and add it to the gitignore).

Reply:
Done.